src/main/resources/application.yaml

```diff
@@ -115,6 +115,7 @@ feign:
     headers:
       - authorization
 userProfUrl: ${USER_PROFILE_URL:http://rd-user-profile-api-aat.service.core-compute-aat.internal}
+locationRefDataUrl: ${LOCATION_REF_DATA_URL:http://rd-location-ref-api-aat.service.core-compute-aat.internal}
```

Comment:
wouldn't LOCATION_REF_DATA_URL will required to be put in values.yaml ?

Reply:
good catch!! will add. thanks